### ISSUE TITLE
[BITV]: change styles for focused items in sidebar

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#41699]
For now it is not really possible to recognize that current item is focused ;(

![Screenshot from 2023-11-23 16-45-46](https://github.com/nextcloud/server/assets/6078378/2e541d79-fe50-4819-868b-0a017d010c3b)
![Screenshot from 2023-11-23 16-45-23](https://github.com/nextcloud/server/assets/6078378/63a778c5-2b36-4641-97b6-966caf80bc26)

@jancborchardt could we do it like in settings? 

![Screenshot from 2023-11-23 16-48-14](https://github.com/nextcloud/server/assets/6078378/6d0820d9-a3d3-4426-a14b-696f6c1cee8c)

**Comment by szaimen** [comment]
It looks like that currently so I guess it is indeed missing a dependency update in server and photos?
![image](https://github.com/nextcloud/server/assets/42591237/1f3eef17-b48b-45cd-83db-e488f572cb30)
![image](https://github.com/nextcloud/server/assets/42591237/654ae6a5-9903-417c-8c5a-fda19eaa8026)

**Comment by szaimen** [comment]
In activity and files dependency update seems to be still misssing
